packages/core/src/components/templates/SearchPage/SearchWrapper.tsx

```diff
@@ -62,6 +64,16 @@ export default function SearchWrapper({
     return <EmptySearch />
   }
 
+  const totalPages = Math.ceil(
+    pageProductGalleryData.search.products.pageInfo.totalCount / itemsPerPage
```

Comment:
suggestion: what do you think about using optional chaining to avoid throwing error?